Makefile

```diff
@@ -1,10 +1,23 @@
-SUBDIRS=kafka-base zookeeper kafka-statefulsets kafka-connect kafka-connect/s2i
+VERSION ?= latest
+
+SUBDIRS=kafka-base zookeeper kafka-statefulsets kafka-inmemory kafka-connect kafka-connect/s2i
 DOCKER_TARGETS=docker_build docker_push docker_tag
 
 all: $(SUBDIRS)
 $(DOCKER_TARGETS): $(SUBDIRS)
+release: release_prepare $(SUBDIRS) release_pkg
+
+release_prepare:
+	rm -rf ./barnabas-$(VERSION)
+	rm -f barnabas-$(VERSION).tar.gz
```

Comment:
`rm -f ./barnabas-$(VERSION).tar.gz` for consistency?

Reply:
Fixed.